newIDE/app/src/UI/Card/CardContent.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import * as React from 'react';
+import MUICardContent from '@material-ui/core/CardContent';
+import { makeStyles } from '@material-ui/core/styles';
+import { marginsSize } from '../Grid';
```

Comment:
are those margins really linked?
Having the same make sense, but I'm wondering if we should define it again here, rather than import a margin used from another component 🤔

Reply:
Alternative is to use Column/Line and/or a bunch of `<Spacer />`?

Reply:
It was suggested to keep consistency between spacings.
Actually I just found the getMuiOverrides in CreateTheme file and it should make more sense to delete those "custom" component and directly override those styles